src/Unity.Mathematics.PerformanceTests/TestHash.cs

```diff
@@ -0,0 +1,49 @@
+using Unity.Burst;
+using Unity.PerformanceTesting;
+using NUnit.Framework;
+using Unity.Collections.LowLevel.Unsafe;
+using System;
+using UnityEngine;
+
+namespace Unity.Mathematics.PerformanceTests
+{
+    [BurstCompile]
+    partial class TestHash
+    {
+        static readonly long kBufferBytes = 1024 * 1024;
+
+        [Test, Performance]
+        public static unsafe void HashAligned()
+        {
+            Debug.Log(Hash(false));
+        }
+
+        [Test, Performance]
+        public static unsafe void HashUnaligned()
+        {
+            Debug.Log(Hash(true));
```

Comment:
I suspect the debug log is to ensure Burst doesn't optimize out the code. Might be good to comment that so it doesn't get removed since some tests check for clean logs which this would potentially break if mathematics tests were ever moved into the same project as another manifest.json `testable`